[2024-01-23T18:41:21.196+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data manual__2024-01-23T18:32:50.376924+00:00 [queued]>
[2024-01-23T18:41:21.229+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data manual__2024-01-23T18:32:50.376924+00:00 [queued]>
[2024-01-23T18:41:21.233+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-23T18:41:21.482+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2024-01-23 18:32:50.376924+00:00
[2024-01-23T18:41:21.496+0000] {standard_task_runner.py:60} INFO - Started process 4088 to run task
[2024-01-23T18:41:21.534+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'manual__2024-01-23T18:32:50.376924+00:00', '--job-id', '484', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmpa61j635a']
[2024-01-23T18:41:21.632+0000] {standard_task_runner.py:88} INFO - Job 484: Subtask get_stackoverflow_data
[2024-01-23T18:41:21.954+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data manual__2024-01-23T18:32:50.376924+00:00 [running]> on host 9924936b75b5
[2024-01-23T18:41:22.551+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-23T18:32:50.376924+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-23T18:32:50.376924+00:00'
[2024-01-23T18:41:25.080+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-23T18:41:25.179+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20240123T183250, start_date=20240123T184121, end_date=20240123T184125
[2024-01-23T18:41:25.298+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T18:41:25.467+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
